roles/libvirt_rhel_vm/tasks/modify_image.yml

```diff
@@ -0,0 +1,26 @@
+- name: upload setup-domain.sh
```

Comment:
Could the [_tempfile_](https://docs.ansible.com/ansible/latest/modules/tempfile_module.html) module be used here?

Reply:
I suppose it could. I left it explicitly named because it's a shell script that we're executing later on and because you can't set mode or ownership with the `tempfile` module. Neither of those things is strictly necessary (the file could just be uploaded and executed by calling "/bin/bash {{ _tmp_file.path }}"), per se.

Reply:
Given that there are other places where _tempfile_ can't be used, I'm not fussed about this anymore.